packages/react-components/src/organisms/SharedResearchAdditionalInformationCard.tsx

```diff
@@ -0,0 +1,157 @@
+import { ResearchOutputResponse } from '@asap-hub/model';
+import { css } from '@emotion/react';
+
+import { Card, Divider, Headline2, Link } from '../atoms';
+import { externalLinkIcon } from '../icons';
+import { mobileScreen, perRem } from '../pixels';
+import { isLink } from '../utils';
+
+const additionalInformationListStyles = css({
+  listStyle: 'none',
+  margin: 0,
+  padding: `${6 / perRem}em 0`,
```

Comment:
Not for this pr, but it would be cool to have a function we call and get the `value + em` like `rem(6)`

Reply:
Yeah, I think that might be a good idea tbh